helpers/filterSeats.js

```diff
@@ -86,8 +111,56 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {
+        'gold': [],
+        'silver': [],
+        'bronze': []
+      };
```

Comment:
@owennicol the band types are not listed in trasformer so currently there is no way to make this object dynamic.

This is an example of what I receive from the server since the last work Tom did:

```
{ 'Grand Circle': { E: 'bronze', D: 'bronze', C: 'silver' },
  'Royal Circle': { B: 'silver', A: 'gold', S: 'gold' },
  Stalls: { D: 'bronze', C: 'silver', B: 'silver', A: 'gold', S: 'gold' } }
```
@TomFirth what do you think?

@sam-cook I discussed with some other developers about this and I agree with you that this helper needs some test coverage but, unfortunately, implementing it now from scratch, even the more basic, would require an amount of work that is beyond the timing of this Jira.
It would be good to split the functionalities of this helper into more isolated and more testable ones as well. I'll make sure that a separated jira will be raised for this as soon as I speak with Claire about when we should implement this.